docs/content/en/concepts/partitions.md

```diff
@@ -0,0 +1,72 @@
+---
+description: >
+  How to create shard partitions
+---
+
+# Partition API
+
+`Partitions` and `PartitionSets` build an API that allows service providers and cluster administrators to create shard Partitions. These partitions can then be leveraged:
+- for the placement of the controllers of the service providers and other components. These APIs provide the information on shard topology required for the scheduling.
```

Comment:
This says `placement` although does not relate to the `Placement` APIs? That could cause some confusion, might be worth to reword a bit?

Reply:
changed to deployment